### ISSUE TITLE
Unable to upload the same picture with behaviour "replace" after deleting it

### ISSUE THREAD

**Issue by MrLexisDev** [dropzone-ui/dropzone-ui-react#268]
**Describe the bug**
I wanna to upload one picture. Then I delete it and then I would like to upload it again.
It doesn't work. I need to upload another picture and then I'll be able to upload the picture I wanted.

**To Reproduce**
1. Go to https://codesandbox.io/s/complete-946hz?file=/src/index.js
2. Click on the dropzone 
3. Upload one picture as for example then try to upload the same picture
4. You can't upload the same picture

**Expected behavior**
I would like to add a picture, then remove it add the same picture.

**Desktop (please complete the following information):**
 - OS: windows 10
 - Browser: chrome
 - Version: 97

**Comment by JinSSJ3** [comment]
Hi @MrLexisDev , 💯 thanks for notifying us you issue. 

We apologige for the inconvenience. 🙏 
I am following the 4 steps listed, It works fine on Firefox.
However, as you described, the problem is on Chrome (actually, Chrome and Edge, since they have the same core).
The problem is that, after an uploading an image (using fakeupload or a real upload), chromium broser somehow recognizes that the image was already uploaded and doesn't allow to add the image to the FileList.
We will take a look in deep and by the end of this day we hope we will have found a way to solve that. 
For now we kindly ask you to work on firefox in the meanwhile.

regards.

**Comment by MrLexisDev** [comment]
Hello and thank you for your fast replying. Unfortunately, I can't force my users to use only firefox. 

But thanks for telling me what's happening with chrome :).

**Comment by JinSSJ3** [comment]
Hi @MrLexisDev, 

Thanks again for notifying us, this helps us to improve the package.
We've just solved the issue. On Chromium based browser like Chrome or Edge it is necessary to explicity clean the value on the input tag, otherwise selecting same file will have no effect and onChage event is not triggered.
We have just also launched 🚀 version 6.5.4 of dropzone-ui 🌌 with this changes. 
So, we kindly ask you to reinstall the package, try out this new version and tell us if this works for you.

regards.
Dropzone-ui team.
